home_audit.node.js

```diff
@@ -900,18 +906,24 @@ let validationRules = {
      * hvac_cooling
      */
     cooling_type_1: function(value) {
-        if(value === 'none' && _homeValues.heating_type_1 === 'none') {
-            return new Validation('Cooling Type is required if there is no Heating Type', ERROR);
-        }
-        return this._cooling_type(value);
+        return this._cooling_type(value, 1);
     },
     cooling_type_2: function(value) {
-        if(value === 'none' && _homeValues.heating_type_2 === 'none') {
+        return this._cooling_type(value, 2);
+    },
+    _cooling_type: function(value, num) {
+    	if(['heat_pump', 'gchp', 'mini_split'].indexOf(value) > -1||
+    	   ['heat_pump', 'gchp', 'mini_split'].indexOf(_homeValues['heating_type_'+num]) > -1)
+    	{
+    		if (value !== _homeValues['heating_type_'+num] &&
+            _homeValues['heating_type_'+num] !== 'none' &&
+            value !== 'none')
+        {
+    			return new Validation('Heating and Cooling Types must match if they are heat pumps.', ERROR);
+    		}
```

Comment:
This is duplicated between heating and cooling, so it should be refactored.

Also, there are tabs in this code instead of spaces.